### ISSUE TITLE
No autocomplete with Twig language mode

### ISSUE THREAD

**Issue by jan-dh** [putyourlightson/vscode-sprig-support#2]
### Describe the bug
I might be mistaken but I can't seem to get this to work on twig language mode.

### To reproduce
Steps to reproduce the behaviour:
1. Set your language mode to HTML (twig)
2. Try to get hints

### Expected behaviour
I'd expect this to work in twig

### Screenshots
<img width="1108" alt="Screenshot 2024-02-25 at 22 11 18" src="https://github.com/putyourlightson/vscode-sprig-support/assets/5801781/1864e448-cdc7-44c5-bd0d-8bb9eb67ea22">

vs

<img width="1120" alt="Screenshot 2024-02-25 at 22 12 50" src="https://github.com/putyourlightson/vscode-sprig-support/assets/5801781/62276b80-4cb2-4a82-bbf1-af85743c10c3">

### Versions
- Plugin version: latest

**Comment by bencroker** [comment]
What is the difference between the first and second screenshot above?

**Comment by jan-dh** [comment]
As mentioned in the title and description, first one has language mode HTML, next one HTML (twig); https://marketplace.visualstudio.com/items?itemName=mblode.twig-language-2 in this case. 

<img width="130" alt="image" src="https://github.com/putyourlightson/vscode-sprig-support/assets/5801781/2e206367-4df5-4edf-a6a4-6c45c6259f5d">

vs 

<img width="198" alt="image" src="https://github.com/putyourlightson/vscode-sprig-support/assets/5801781/4547adcb-7bab-431c-8e9c-9cc3a3c8e93b">


Maybe adding some info to the docs to enable this for other languages besides html would be enough.
I assume this is because you explicitly set this as HTML here?

```
  "contributes": {
        "html": {
            "customData": [
                "./src/sprig-support.json"
            ]
        }
    },
```

**Comment by bencroker** [comment]
Typing `s-` in a Twig file brings up autocomplete suggestions for me. Is this not the case for you?

![CleanShot 2024-02-26 at 06 53 23@2x](https://github.com/putyourlightson/vscode-sprig-support/assets/57572400/6f8c2d58-234f-4e74-8880-3d3eaa1684c9)

**Comment by jan-dh** [comment]
No

<img width="241" alt="image" src="https://github.com/putyourlightson/vscode-sprig-support/assets/5801781/56458a12-b182-43ff-b08f-abfda3d118d9">

**Comment by bencroker** [comment]
Ok now I can reproduce. It doesn’t help that I don’t actually use VSCode as my IDE!

It probably does require adding `twig` to the `contributes` property, yes. I wonder if I should be using the `grammars` property, though, as I am not seeing `contributes.html` in the docs at https://code.visualstudio.com/api/references/contribution-points#contributes.grammars

**Comment by bencroker** [comment]
Just released version 1.0.7 with explicit support for Twig, but I have a feeling that the language mode must be set to `HTML` for it to work. I’m not sure what would be required to make it work in the `HTML (Twig)` mode, probably better overall support for Twig in VSCode, unfortunately.
